pyblish_qml/host.py

```diff
@@ -277,7 +277,11 @@ def install_host(use_threaded_wrapper):
 
 def install_event_filter():
 
-    main_window = _state["vesselParent"]
+    main_window = _state.get("vesselParent")
+    if main_window is None:
+        sys.stdout.write("Main window not found, event filter did not "
```

Comment:
I understand this changes nothing about the code logic and is kind of insignificant, but if you want to be more grammatically correct about "did not installed", this can be better written as one of two ways:
- `Main window not found, event filter did not get installed.`
- `Main window not found, event filter did not install.`